rfc/wip-tform/index.md

```diff
@@ -0,0 +1,679 @@
+  <dd> 	The scale parameters stored as a JSON list of numbers. The list MUST have length `N`.</dd>
+</dl>
+
+
+#### <a name="affine">affine</a>
+
+`affine`s are [matrix transformations](#matrix-transformations) from N-dimensional inputs to M-dimensional outputs are represented at `(N)x(M+1)`
+matrices in homogeneous coordinates. This transformation type may be (but is not necessarily) invertible when `N` equals `M`.
+The matrix MUST be stored as a 2D array either as json or in a zarr array.
+
+<dl>
+  <dt><strong>path</strong></dt>
+  <dd>  The path to a zarr-array containing the affine parameters.
+	The array at this path MUST be 2D whose shape MUST be `N x (M+1)`.</dd>
+  <dt><strong>affine</strong></dt>
+  <dd> 	The affine parameters stored in JSON. The matrix MUST be stored as 2D nested array where the outer array MUST be length
+  `N` and the inner arrays MUST be length `M+1`.</dd>
+</dl>
+
+
+#### <a name="rotation">rotation</a>
+
+`rotation`s are [matrix transformations](#matrix-transformations) that are special cases of affine transformations. When possible, a rotation
+transformation SHOULD be preferred to its equivalent affine. Input and output dimensionality (N) MUST be identical. Rotations
+are stored as `NxN` matrices, see below, and MUST have determinant equal to one, with orthonormal rows and columns. The matrix
+MUST be stored as a 2D array either as json or in a zarr array. `rotation` transformations are invertible.
+
+<dl>
+  <dt><strong>path</strong></dt>
+  <dd>  The path to an array containing the affine parameters.
+	The array at this path MUST be 2D whose shape MUST be `N x N`.</dd>
+  <dt><strong>rotation</strong></dt>
+  <dd> 	The  parameters stored in JSON. The matrix MUST be stored as a 2D nested array where the outer array MUST be length `N`
+  and the inner arrays MUST be length `N`.</dd>
+</dl>
+
+
+#### <a name="inverseOf">inverseOf</a>
+
+An `inverseOf` transformation contains another transformation (often non-linear), and indicates that
+transforming points from output to input coordinate systems is possible using the contained transformation.
+Transforming points from the input to the output coordinate systems requires the inverse of the contained
+transformation (if it exists).
+
+<div class=note>
+    Software libraries that perform image registration often return the transformation from fixed image
+    coordinates to moving image coordinates, because this "inverse" transformation is most often required
+    when rendering the transformed moving image. Results such as this may be enclosed in an `inverseOf`
+    transformation. This enables the "outer" coordinate transformation to specify the moving image coordinates
+    as `input` and fixed image coordinates as `output`, a choice that many users and developers find intuitive.
+</div>
+
+
+
+#### <a name="sequence">sequence</a>
+
+A `sequence` transformation consists of an ordered array of coordinate transformations, and is invertible if and only if every
```

Comment:
I'd rewrite the text to the following:

A `sequence` transformation consists of an ordered array of coordinate transformations. If all the coordinate transformations of the sequence are invertible, then the sequence transformation is also invertible. The opposite is not always true, but we add the constrain that all the transformations need to be invertible, because implementations may choose to apply a sequence transformation by applying the transformations one at the time.

Asking for feedback @d-v-b wdyt? Not adding this constraint would make implementations more complex because we can't apply a recursive logic to invert a sequence transformation.

Reply:
Thanks, I clarified the text a little (see below)

### the new text
> A `sequence` transformation consists of an ordered array of coordinate transformations, and is invertible if every coordinate transform in the array is invertible (though could be invertible in other cases as well). 

compare to
####  the old text
> A sequence transformation consists of an ordered array of coordinate transformations, and is invertible if and only if every coordinate transform in the array is invertible.